tests/cstor/gtest/test_zrepl_prot.cc

```diff
@@ -296,6 +296,42 @@ static void get_zvol_status(std::string zvol_name, uint64_t &ioseq, int control_
 	EXPECT_EQ(status.rebuild_status, rebuild_status);
 }
 
+static void wait_for_replica_status(std::string zvol_name, uint64_t &ioseq,
+    int control_fd, int state, struct zrepl_status_ack &status,
+    int version = REPLICA_VERSION) {
+	zvol_io_hdr_t hdr_in, hdr_out = {0};
+	int rc, max_retry_count=10, retry_count = 0;
+
+	while (retry_count < max_retry_count) {
+		hdr_out.version = version;
```

Comment:
this looks duplicate code. Please use get_zvol_status for getting the status.

Reply:
I will reuse it

Reply:
Done